src/scenes/CreateEvent/Outcomes.js

```diff
@@ -43,7 +44,7 @@ const Button = styled(_Button).attrs({ variant: 'raised' })`
   width: 150px;
 `;
 
-const Outcome = injectIntl(observer(({ outcome, createEvent, i, intl }) => (
+const Outcome = injectIntl(observer(withStyles(styles, { withTheme: true })(({ classes, outcome, createEvent, i, intl }) => (
```

Comment:
`observer` should be the last thing injected. move withStyles before `observer` or `injectIntl`. sometimes this can cause weird bugs with Mobx.